helpers/leaderboard.js

```diff
@@ -1,3 +1,7 @@
+/*
+ * This gives linting error, and since it is not used, it is commented out.
+ *
```

Comment:
Delete?

Reply:
We will keep it for next year and explain why in README